src/reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({
```

Comment:
- [optional] please kindly remove boilerplate files created while creating the react app so your work can be more clean .. like
index.css
reportwebvitals.js